### ISSUE TITLE
Evaluation code and data

### ISSUE THREAD

**Issue by ramacha7** [mit-han-lab/fastcomposer#21]
Hi, I was wondering when the evaluation code and data will be updated on the github?

**Comment by tianweiy** [comment]
thank you for the interest. we are working on a revision and it will probably be around mid-october for the new release.

**Comment by ramacha7** [comment]
Thank you!

**Comment by Jiawei-Yao0812** [comment]
Hi Tianwei, thanks for your excellent repo. I would like to know if you still have any plans to open source the eval code around mid-october, and if so, do you have a specific time for releasing the evaluation code? Thank you so much for your help.

**Comment by tianweiy** [comment]
thank you for the reminder. I will find them to prepare them this week!

**Comment by ramacha7** [comment]
Hi Tianwei! Would it be possible if you could release the evaluation code whenever you could? Thank you very much in advance

**Comment by tianweiy** [comment]
Yes, yes. Sorry, I have been too busy recently. I will push them as soon as I finish. Thank you for waiting

**Comment by tianweiy** [comment]
Yes, yes. Sorry, I have been too busy recently. I will push them as soon as I finish. Thank you for waiting

**Comment by sophistz** [comment]
Hi, I'm wondering which "dreamer" package are you using for evaluation. I tried to install it by pip but it seems not to be the correct one.

**Comment by malteprinzler** [comment]
is there any news on this? I would also like to reproduce the numbers from the paper
